src/components/chip/chip.scss

```diff
@@ -103,6 +103,40 @@
     }
 }
 
+:host(limel-chip[type='filter']) {
```

Comment:
this is an intentional choice, and gives us a new UI 👇 
<img width="299" alt="image" src="https://github.com/Lundalogik/lime-elements/assets/35954987/580bd838-2e24-4c37-9000-5774ed67dc6c">


which will replace this old UI 👇  when we implement this component in the chip-set
<img width="329" alt="image" src="https://github.com/Lundalogik/lime-elements/assets/35954987/72754ced-2a89-4d93-8e65-74c16755397e">

Reply:
Gotta say, this new filter style looks great, and the transition when selecting and deselecting the chips is _beautiful_! Great work! 🙌